### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by johnmccombs1**
# Description
Update README.md

Fixes: (link Jira ticket)

#changelog Update README.md

# Type of change

- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
Please validate that you have tested your change in at least one of the following areas:

- [ ] Successfully tested locally (or docker image)
- [ ] Successfully tested on a staging or sandbox server
- [ ] Successfully tested on a merchant's staging server

# For PR Reviewer 
- [ ] Reviewed unit tests to make sure we are using real components rather than mocks as much as possible?
- [ ] For any major change (observer, new Bolt feature, core Magento interaction) we must add a feature switch, did you verify this?

# Checklist:

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have created or modified unit tests to sufficiently cover my changes.
- [ ] I have added my Jira ticket link and provided a changelog message.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,8 +33,9 @@ Select ***Yes*** to enable it.
 + Enter an appropriate `Title` such as ***Credit & Debit Cards***
 
 > #### API credentials
-> The following four required values, (i.e. `API Key`, `Signing Secret`, `Publishable Key - Multi Step`, and `Publishable Key - Payment Only` can be found in your ***Bolt Merchant Dashboard***  under `Settings` > `Users and Keys`
->
+> The following four required values, (i.e. `API Key`, `Signing Secret`, `Publishable Key ` can be found in your ***Bolt Merchant Dashboard***  under `Developers` > `API`
```
Comment (njfaries) [inline README.md:36]: nit: missing closing parenthesis

Comment (njfaries) [reply README.md:36]: this is now three, right? with the other publishable keys being elsewhere

File: README.md [hunk 2]
```diff
@@ -33,8 +33,9 @@ Select ***Yes*** to enable it.
...
+> The Publishable Key is for the multi-step checkout by default. You can find other publishable keys (Payment Only and Back-Office) in the division dropdown at the top-right concern.
```
Comment (njfaries) [inline README.md:37]: top-right corner?

**Review by njfaries — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (johnmccombs1) [reply README.md]: @njfaries Sorry, my typo :) I've updated it.

File: README.md (hunk 1 above)
Comment (johnmccombs1) [reply README.md]: @njfaries 
nit: missing closing parenthesis
=> I've updated it

this is now three, right? with the other publishable keys being elsewhere
=> Yes

File: README.md [hunk 3]
```diff
@@ -33,8 +33,9 @@ Select ***Yes*** to enable it.
 + Enter an appropriate `Title` such as ***Credit & Debit Cards***
 
 > #### API credentials
-> The following four required values, (i.e. `API Key`, `Signing Secret`, `Publishable Key - Multi Step`, and `Publishable Key - Payment Only` can be found in your ***Bolt Merchant Dashboard***  under `Settings` > `Users and Keys`
->
+> The following four required values, (i.e. `API Key`, `Signing Secret`, `Publishable Key `) can be found in your ***Bolt Merchant Dashboard***  under `Developers` > `API`
```
Comment (adentong) [inline README.md:36]:
```suggestion
> The following three required values, (i.e. `API Key`, `Signing Secret`, `Publishable Key`) can be found in your ***Bolt Merchant Dashboard***  under `Developers` > `API`
```

**Review by vitaliyreznikov — APPROVED** [review]

**Review by danac-gs — APPROVED** [review]

**Review by adentong — APPROVED** [review]

File: README.md (hunk 3 above)
Comment (njfaries) [reply README.md:36]: This still hasn't been committed.

**Review by njfaries — CHANGES_REQUESTED** [review]

**Review by njfaries — APPROVED** [review]

**Pull request merged**